source/js/payments-card.js

```diff
@@ -51,6 +63,50 @@ function setupBraintree() {
     container.classList.remove(braintreeErrorClass);
   }
 
+  function showDetailFieldError(field) {
+    // If field error is already showing, do not render again
+    if (!field.parentNode.classList.contains("form-item--errors")) {
+      field.parentNode.classList.add("form-item--errors");
+      document
+        .getElementById(`error-message__` + `${field.name}`)
+        .classList.remove("hidden");
+    }
+  }
+
+  function checkYourDetailsFields() {
+    // Looping through required "Your Details" fields, in order to prevent a bad form submit,
+    // which will result in CC data loss.
+
+    var yourDetailsFormValidity = true;
+
+    for (const [fieldKey, inputElement] of Object.entries(requiredFields)) {
+      // Since postcode is not always rendered and needed, we need a special case for it.
+      if (fieldKey == "postCodeInput") {
+        // If Post Code Field is rendered, verify its filled out.
+        if (
+          !inputElement.parentNode.parentNode.classList.contains("hidden") &
```

Comment:
Hi @fessehaye! I was wondering if you knew any other way to preform this line as "parentNode.parentNode" does not feel like the best way to do this, but it works!

Pretty much what I am trying to do is see if the div that houses the "postal code" input element has the "hidden" class, however it is 2 levels up

Thanks in advance!